libsolidity/formal/BMC.cpp

```diff
@@ -118,13 +118,18 @@ void BMC::analyze(SourceUnit const& _source, std::map<ASTNode const*, std::set<V
 		);
 
 	if (!m_settings.showProvedSafe && !m_safeTargets.empty())
+	{
+		std::size_t provedSafeNum = 0;
+		for (auto e: m_safeTargets)
+			provedSafeNum += e.second.size();
```

Comment:
In the current version, you would be making a copy of each entry in `m_safeTargets`, no?

```suggestion
		for (auto&& [_, targets]: m_safeTargets)
			provedSafeNum += targets.size();
```